projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,222 @@
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.firstDate | cxDate }}</a
+                >
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.frequency' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.trigger.displayTimeTable }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.nextOrderDate' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.trigger.activationTime | cxDate }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.total' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.subTotal.formattedValue }}</a
+                >
+              </td>
+              <td (click)="openDialog()">
+                <a
+                  (click)="openDialog()"
+                  class="cx-order-cancel btn btn-link"
+                  #element
+                >
+                  {{ 'orderHistory.cancel' | cxTranslate | uppercase }}
+                </a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <!-- Select Form and Pagination Bottom -->
+        <div class="cx-replenishment-order-history-sort bottom row">
```

Comment:
I see that when you removed the extra div that needed to remove, you updated the content to flex-end instead of space-between which is good, however you haven't tackled the issue, where it lost it's paddings/margin (whichever was used before

![Screen Shot 2020-09-23 at 3 51 05 PM](https://user-images.githubusercontent.com/29099466/94062398-9d1bb200-fdb4-11ea-933d-7db3d10a9e84.png)